selftests/checkall

```diff
@@ -33,8 +33,9 @@ parallel_selftests() {
     for I in $(seq 0 $PER_SLICE $((${#ALL[@]} - 1))); do
         TMP=$(mktemp /tmp/avocado_parallel_unittest_output_XXXXXX)
         TMPS+=("$TMP")
-        python -m unittest ${ALL[@]:$I:$PER_SLICE} &> $TMP &
+        ( python -m unittest ${ALL[@]:$I:$PER_SLICE} &> $TMP ) &
         PIDS+=("$!")
+        sleep 0.1
```

Comment:
The only thing that seems to make difference here is the last `sleep` before the `wait`. I'd just add a `sleep 0.1` before      the `for I in $(seq 0 $((${#PIDS[@]} - 1))); do` and see if the issue still happens (it won't for me).

Reply:
Sure, I wasn't sure why it's failing and this sleep worked well for me (even though I don't understand why). Feel free to send patches 😄